srcpkgs/wireplumber/files/wireplumber/control/t

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/usr/bin/kill -TERM $(pgrep -P $(cat /run/runit/supervise.wireplumber/pid) | tail -1)
```

Comment:
What are you trying to accomplish here? This seems likely to be indeterminate, sending the term to some arbitrary child of wireplumber.

Reply:
dbus-run-session, for some reason [doesn't kill its child](https://gitlab.freedesktop.org/dbus/dbus/-/issues/389) `dbus-daemon` and the command invoked atm, might be a bug, as the parent who is expected to monitor the daemon should kill the child before it exits, `dbus-launch` does this, but the daemon doesn't exits if underlying command exits.

This kills the wireplumber (last fork).

Reply:
In that case, try adding `-P` to `chpst` to start the session in its own process group, then you should be able to replace this line with
```sh
[ -r supervise/pid ] && kill -TERM -$(cat supervise/pid)
```
Adding a `-` before the PID will signal the entire process group created by `chpst`. Use the relative path `supervise/pid` because the script shouldn't directly know about anything outside of its directory, and drop the full path to `kill`.